tools/convert_library.py

```
@@ -837,6 +841,9 @@ def build_ids_set(tab_name):
     library_vars_dict["tab"][x] for x in library_vars_dict["tab"]
 ]
 
+lib_date = library_vars.get("library_publication_date", datetime.datetime.now())
+if type(lib_date) == datetime.datetime:
```

Ruff (0.8.2)

Use `is` and `is not` for type comparisons, or `isinstance()` for isinstance checks

(E721)